packages/playground/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "private": false,
+  "name": "@cognite/sdk-playground",
+  "license": "Apache-2.0",
+  "repository": "cognitedata/cognite-sdk-js",
+  "homepage": "https://github.com/cognitedata/cognite-sdk-js#readme",
+  "main": "dist/index.js",
+  "types": "dist/src/index.d.ts",
+  "version": "3.0.2",
+  "scripts": {
+    "clean": "rm -rf dist/ docs/ codeSnippets/",
+    "test": "jest --config=../../jest.config.js --testPathPattern=/playground/",
+    "lint": "eslint 'src/**/*.{js,ts}'",
+    "lint:fix": "yarn lint --fix",
+    "prepublishOnly": "yarn build",
+    "build": "yarn clean && rollup -c && yarn esCheck",
+    "watch": "rollup -cw",
+    "esCheck": "es-check es5 './dist/index.js'",
+    "docs": "typedoc --options typedoc.js --tsconfig tsconfig.json src/index.ts",
+    "docs:bundle": "yarn docs && mkdir -p ../../docs/playground && cp -r docs/* ../../docs/playground/",
+    "extract-snippets": "rm -rf codeSnippets/ && yarn docs --json codeSnippets/docs.json && node ../../scripts/extractCodeSnippets.js --package-name @cognite/sdk-playground",
+    "test-snippets": "yarn extract-snippets && yarn tsc -p codeSnippets/tsconfig.build.json"
+  },
+  "dependencies": {
+    "@cognite/sdk": "^4.3.0",
```

Comment:
you are not actually using it? it seems like you only need core

Reply:
I use it in the types.ts file